src/js/view/controls/controls.js

```diff
@@ -197,10 +197,22 @@ export default class Controls {
                 // Let event bubble upwards
                 return true;
             }
-            // On keypress show the controlbar for a few seconds
+
+            // keep controls active when navigating inside the player
             if (!this.instreamState) {
-                this.userActive();
+                const container = this.playerContainer;
+                const fullscreenButton = this.controlbar.elements.fullscreen;
+                const insideContainer = container !== evt.target && container.contains(evt.target);
+                const isTab = evt.keyCode === 9;
+
+                // check if we tab after fullscreen or shift-tab from player
+                const exitingContainer =
+                    (isTab && evt.shiftKey && evt.target === container) ||
+                    (isTab && evt.target === fullscreenButton.element());
+
+                this.userActive(null, insideContainer && !exitingContainer);
```

Comment:
This should be done on key up because then the tabbing would be done and we only need to check `insideContainer`.